hashing/te-tag-query-python/all_in_one_lib/commands/match.py

```diff
@@ -0,0 +1,139 @@
+##!/usr/bin/env python
```

Comment:
this file gave me more squint than the rest ... i'd love some explanatory/narrative comments to walk the reader through what it's doing

Reply:
Yea, need to go through the whole thing and add:

1. Docstrings for every file
2. Long descriptions for every command